src/cljs/rems/application.cljs

```diff
@@ -788,11 +781,11 @@
                (when state (str ": " (localize-state state))))]
       :always [:div
                [:div.mb-3 {:class (str "state-" (if (keyword? state) (name state) state))} (phases phases-data)]
-               (when latest-comment
-                 (info-field (text :t.applications/latest-comment)
-                             latest-comment))]
-      :collapse (when (seq event-groups)
-                  [events-view event-groups])}]))
+               [:h4 (text :t.form/events)]
+               (when-let [g [(first event-groups)]]
```

Comment:
Should be `(when-let [g (first event-groups)]`

Reply:
Well, then need to create the seq in the next line, but fine.